src/WPF/ArcGISRuntime.WPF.Viewer/Samples/GeometryEngine/SpatialRelationships/SpatialRelationships.xaml.cs

```diff
@@ -0,0 +1,176 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ArcGISRuntime.WPF.Samples.SpatialRelationships
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Spatial relationships",
+        "GeometryEngine",
+        "This sample demonstrates how to use the GeometryEngine to evaluate the spatial relationships (for example, polygon a contains line b) of between geometries.",
+        "Tap a graphic to select it. The display will update to show the relationships with the other graphics.")]
+    public partial class SpatialRelationships
+    {
```

Comment:
Add code comments for the member variables in the WPF, UWP, and Xamarin.Forms projects. In Android and iOS, the following code comment is available:
```
        // References to the graphics and graphics overlay
```